server/views/pages/overview/partials/educationAndTrainingTab/educationAndTrainingTabContents.njk

```diff
@@ -10,15 +10,59 @@ The 'Training and eduction in prison' summary card is only shown if the short qu
 {% include './_functionalSkills.njk' %}
 {% include './_inPrisonQualifications.njk' %}
 
-{% if educationAndTraining.inductionQuestionSet == 'LONG_QUESTION_SET' %}
-  <section data-qa="qualifications-and-education-history-long-question-set">
-    {% include './_qualificationsAndEducationHistory_inductionLongQuestionSet.njk' %}
-  </section>
+
```

Comment:
We could probably improve this by moving some/all of it out into some other include files ... but I've done it this way for now in order to reduce the risk of merge conflicts with the stuff that @chrimesdev is currently doing
(ie. Adam will likely be uncommenting the include on line 26 soon - I had moved the whole lot out into another file it would cause merge conflict hell!)
We can look at tidying it once both Adam and I are finished in this general area 👍